plugins/cfhour.js

```diff
@@ -85,9 +87,9 @@ module.exports = (function(){
 				}
 				else {
 					console.error(error);
+					onErrorDebounce(error);
```

Comment:
I had that same thought while going over this initially, but I think I got distracted before I could write it down.